src/NServiceBus.Core/Routing/AutomaticSubscriptions/AutoSubscribe.cs

```diff
@@ -87,6 +89,7 @@ public ApplySubscriptions(IEnumerable<Type> eventsToSubscribe, IBus bus)
                 this.bus = bus;
             }
 
+            // This is solved when the features receive a context
```

Comment:
Doesn't need DI to resolve `IBus` since we are passing in the context in https://github.com/Particular/NServiceBus/pull/3053